### DISCUSSION TITLE
Stuck on Step "Deploy the L1 contracts"

### DISCUSSION THREAD

**Discussion by yusufsuhair** [ethereum-optimism/developers#354] [Deploying the OP Stack]
Hi All,

I was creating my own L2 Rollup testnet, at first I did that successfully on my laptop (Macbook) which I can literally use the created L2 network. 

So then I purchased a VM using Ubuntu v22 OS and tried to deploy referring to the doc (https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#proposer).

But then I was stuck in this section:
![image](https://github.com/ethereum-optimism/docs/assets/53525901/ffae09f4-8e9e-40e1-9274-a3f979f8f4e7)

The error mentioned was:

`...16060820152818360808301376000818301608090810191909152601f9092017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe01601019392505050565b838152818360208301376000910160200190815292915050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fdfea164736f6c634300080f000a496e697469616c697a61626c653a20636f6e7472616374206973206e6f742069\",\n    \"nonce\": \"0x4a\",\n    \"accessList\": []\n  },\n  \"additionalContracts\": [],\n  \"isFixedGasLimit\": false\n}'"])
    │   └─ ← [Return] 0x6e756c6c
    ├─ [0] VM::ffi(["bash", "-c", "echo DisputeGameFactory.0.8.15 | sed -E 's/[.][0-9]+\\.[0-9]+\\.[0-9]+//g'"])
    │   └─ ← [Return] 0x4469737075746547616d65466163746f7279
    ├─ [0] VM::getCode("DisputeGameFactory.sol:DisputeGameFactory.0.8.15") [staticcall]
    │   └─ ← [Revert] No matching artifact found
    └─ ← [Revert] No matching artifact found`

`== Logs ==
  Connected to network with chainid 11155111
  Storing temp deployment data in /root/optimism/packages/contracts-bedrock/deployments/getting-started/.deploy
  DeployConfig: reading file /root/optimism/packages/contracts-bedrock/deploy-config/getting-started.json
  Deploying from Deploy
  Deployment context: getting-started
  Syncing 27 deployments
  Using deployment artifact /root/optimism/packages/contracts-bedrock/broadcast/Deploy.s.sol/11155111/run-latest.json
  Syncing deployment AddressManager: contract AddressManager
  Syncing deployment DisputeGameFactory: contract DisputeGameFactory.0.8.15
Error:
script failed: No matching artifact found`

I have no idea whats wrong, I even tried to create another VM using the Ubuntu v20 instead, but same issue..

and here is the versions after running the `versions.sh`:
![image](https://github.com/ethereum-optimism/docs/assets/53525901/d226b615-b621-4ca3-bc58-088ce93235cb)

My suspicion was at the `make v4.x.x` instead of `make v3` which my MacBook uses `make v3`, it was because ubuntu doesn't allow to use `make v3` anymore. But I don't think that's gonna be an issue. Kindly advise ..

However, I do believe that all of the versions are comply

Please assist

**Comment by cpengilly** [comment]
@yusufsuhair apologize for the delay, I have notifed our devsupport team of this issue and we should have an answer or workaround in the next few days!

**Reply by yusufsuhair** [reply]
Okay I will firmly waiting

**Comment by sbvegan** [comment]
Hey @yusufsuhair, I went ahead and moved your question into the place where it'll have the most eyes. I can't take a look at the moment, but someone else might be able to hop in and answer.

**Comment by opfocus** [comment]
https://github.com/ethereum-optimism/optimism/issues/10114

**Reply by opfocus** [reply]
tutorials/chain  -- Are you Creating Your Own L2 Rollup Testnet in this branch？

**Reply by yusufsuhair** [reply]
Yep, correct, I used exactly that branch

**Reply by yusufsuhair** [reply]
I believe the discussion link above you sent was an another issue yeah?

**Reply by opfocus** [reply]
I see , I will have try

**Reply by yusufsuhair** [reply]
Hey, perhaps you wanna try login into my ubuntu VM ? 

and try run the script there, I am still stuck at that deploy part

**Reply by opfocus** [reply]
https://github.com/ethereum-optimism/docs/issues/627   We are discussing it, and some links include solutions, I think.

**Reply by yusufsuhair** [reply]
Okay thanks!

BTW I tried with 2 ubuntu versions:

- Ubuntu 23.10
- Ubuntu 22.04.4 LTS

so current i am using CentOS 9 Stream x64, still same issue

hope that helps!

**Reply by opfocus** [reply]
"If you're still stuck at that point in the log, you can try skipping the forge script scripts/Deploy.s.sol:Deploy --sig 'sync()' --rpc-url $L1_RPC_URL,  execute 
```
go run cmd/main.go genesis l2 \
--deploy-config ../packages/contracts-bedrock/deploy-config/getting-started.json \
--l1-deployments ../packages/contracts-bedrock/deployments/getting-started/.deploy \
--outfile.l2 genesis.json \
--outfile.rollup rollup.json \
--l1-rpc $L1_RPC_URL
```

I haven't tried it because I wasn't successful in reproducing the error you encountered. I believe the solution above is likely viable (before executing, you can check if the .deploy file contains the required addresses: vim ~/optimism/packages/contracts-bedrock/deployments/getting-started/.deploy)

**Reply by yusufsuhair** [reply]
hey it works, now I can run op-batcher, op-geth, op-node , but except op-proposer i cant because it said that the L2OutputOracleProxy.json wasn't there 

`./bin/op-proposer \
  --poll-interval=12s \
  --rpc.port=8560 \
  --rollup-rpc=http://localhost:8547 \
  --l2oo-address=$(cat ../packages/contracts-bedrock/deployments/getting-started/L2OutputOracleProxy.json | jq -r .address) \
  --private-key=$GS_PROPOSER_PRIVATE_KEY \
  --l1-eth-rpc=$L1_RPC_URL
cat: ../packages/contracts-bedrock/deployments/getting-started/L2OutputOracleProxy.json: No such file or directory
INFO [04-17|11:26:49.250] Initializing L2Output Submitter
INFO [04-17|11:26:49.395] metrics disabled
INFO [04-17|11:26:49.395] Stopping Proposer
INFO [04-17|11:26:49.395] L2Output Submitter stopped
CRIT [04-17|11:26:49.395] Application failed                       message="failed to setup: failed to init Driver: neither the `L2OutputOracle` nor `DisputeGameFactory` addresses were provided"`

Maybe it was because of the skipped part (`forge script scripts/Deploy.s.sol:Deploy --sig 'sync()' --rpc-url $L1_RPC_URL`)

However, I tried again to deploy that skipped part, still same error :S

`...000000000000000000000000000000000000000000000000600052603260045260246000fdfea164736f6c634300080f000a496e697469616c697a61626c653a20636f6e7472616374206973206e6f742069\",\n    \"nonce\": \"0x79\",\n    \"accessList\": []\n  },\n  \"additionalContracts\": [],\n  \"isFixedGasLimit\": false\n}'"])
    │   └─ ← [Return] 0x6e756c6c
    ├─ [0] VM::ffi(["bash", "-c", "echo DisputeGameFactory.0.8.15 | sed -E 's/[.][0-9]+\\.[0-9]+\\.[0-9]+//g'"])
    │   └─ ← [Return] 0x4469737075746547616d65466163746f7279
    ├─ [0] VM::getCode("DisputeGameFactory.sol:DisputeGameFactory.0.8.15") [staticcall]
    │   └─ ← [Revert] No matching artifact found
    └─ ← [Revert] No matching artifact found`



`== Logs ==
  Connected to network with chainid 11155111
  Storing temp deployment data in /root/optimism/packages/contracts-bedrock/deployments/getting-started/.deploy
  DeployConfig: reading file /root/optimism/packages/contracts-bedrock/deploy-config/getting-started.json
  Deploying from Deploy
  Deployment context: getting-started
  Syncing 27 deployments
  Using deployment artifact /root/optimism/packages/contracts-bedrock/broadcast/Deploy.s.sol/11155111/run-latest.json
  Syncing deployment AddressManager: contract AddressManager
  Syncing deployment DisputeGameFactory: contract DisputeGameFactory.0.8.15
Error:
script failed: No matching artifact found`

==============

also maybe I cant skip that step because I need to send ETH to the L1StandardBridgeProxy 

![image](https://github.com/ethereum-optimism/developers/assets/53525901/5c1d305f-c11f-444b-9cba-32cbdaf52598)

Error here because it wasn't deployed perhaps

`cat deployments/getting-started/L1StandardBridgeProxy.json | jq -r .address`
`cat: deployments/getting-started/L1StandardBridgeProxy.json: No such file or directory`

**Reply by opfocus** [reply]
Yes, since we skipped that step, we don't have the L2OutputOracleProxy.json, but we can easily find the address in .deploy and manually fill in the address --l2oo-address.     --l2oo-address=0x.... \

**Reply by opfocus** [reply]
Manually filling in the address is just a temporary solution (this includes any required addresses, rather than automatically reading them via terminal commands). We're waiting for the final solution.

**Reply by yusufsuhair** [reply]
Okay i just less the `.deploy` , and there was a L1StandardBridgeProxy wallet address there, I send the ETH to there right?

**Reply by opfocus** [reply]
You can't proceed right now because your op stack chain needs to synchronize for a while. Wait for the synchronization to complete, then the proposer and batcher will work normally. This usually happens quickly   https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-8570927

**Reply by yusufsuhair** [reply]
okay saw that discussion to check weather those are up n running or not,

do you know where is the logs saved at?

**Reply by opfocus** [reply]
In the absence of using Docker, the default behavior is to output to the terminal.

**Reply by yusufsuhair** [reply]
after waiting

im getting this issue on op-proposer

`WARN [04-17|16:53:10.377] Failed to create a transaction, will retry service=proposer err="failed to estimate gas: execution reverted: L2OutputOracle: block hash does not match the hash at the expected height"`

idk was this blocking or not, but I just cant reach the RPC from my metamask

**Reply by yusufsuhair** [reply]
But i also did have this on the op-node

`INFO [04-17|16:54:10.262] Sync progress                            reason="reconciled with L1" `

**Reply by opfocus** [reply]
Is your op-geth log working properly? It should look something like this:
```
INFO [04-17|17:31:36.104] Starting work on payload                 id=0x0256df2cbd5e863e
INFO [04-17|17:31:36.105] Updated payload                          id=0x0256df2cbd5e863e number=4435 hash=119187..cb31da txs=1 withdrawals=0 gas=50477 fees=0 root=3578f9..f624b4 elapsed="709.303µs"
INFO [04-17|17:31:38.104] Stopping work on payload                 id=0x0256df2cbd5e863e reason=timeout elapsed=2000
INFO [04-17|17:36:37.524] Imported new potential chain segment     number=4435 hash=119187..cb31da blocks=1 txs=1 mgas=0.050 elapsed=12.594ms mgasps=4.008 age=4m59s snapdiffs=852.21KiB triedirty=0.00B
INFO [04-17|17:36:37.531] Chain head was updated                   number=4435 hash=119187..cb31da root=3578f9..f624b4 elapsed=4.713743ms age=4m59s

```

**Reply by yusufsuhair** [reply]
yep, it is working properly

**Reply by yusufsuhair** [reply]
![image](https://github.com/ethereum-optimism/developers/assets/53525901/701c244b-bfa1-4a6f-8cc1-017b42e149ec)

from the picture above, here is the sequence from the terminal:

op-geth | op-node | op-batcher | op-proposer

**Reply by yusufsuhair** [reply]
hi all, any finding for this perhaps? TQ :)

**Reply by opfocus** [reply]
Please paste the complete log entry .INFO [04-17|16:54:10.262] Sync progress                            reason="reconciled with L1"

**Reply by yusufsuhair** [reply]
Sorry I didnt get you, my answer previously wasnt complete?

![image](https://github.com/ethereum-optimism/developers/assets/53525901/5a1d608e-4295-42c8-9c9a-545c05482b30)

**Reply by opfocus** [reply]
This is not a complete log message, is it?

**Reply by opfocus** [reply]
INFO [04-19|17:18:08.383] Sync progress                            reason="reconciled with L1"                   l2_finalized=0aab6a..447419:67613 l2_safe=fd20a2..d04265:68158 l2_pending_safe=fd20a2..d04265:68158 l2_unsafe=b36fc5..35bc39:90430 l2_time=1,713,547,088 l1_derived=9e9fb5..208a51:5733062

**Reply by yusufsuhair** [reply]
hmm okay say it wasnt complete

can you help me find where the log saved?

**Reply by opfocus** [reply]
If you follow the tutorial completely, it will output in the terminal. Because closing the terminal will cause the component to stop running, I used tmux to run them in the background

**Reply by yusufsuhair** [reply]
okay let me try on tmux,

so first of all, the first error I found was inside `op-proposer` which are like so:

`WARN [04-19|18:24:23.542] Failed to create a transaction, will retry service=proposer err="failed to estimate gas: execution reverted: L2OutputOracle: block hash does not match the hash at the expected height"
WARN [04-19|18:24:25.830] Failed to create a transaction, will retry service=proposer err="failed to estimate gas: execution reverted: L2OutputOracle: block hash does not match the hash at the expected height"
WARN [04-19|18:24:28.122] Failed to create a transaction, will retry service=proposer err="failed to estimate gas: execution reverted: L2OutputOracle: block hash does not match the hash at the expected height"
WARN [04-19|18:24:30.412] Failed to create a transaction, will retry service=proposer err="failed to estimate gas: 429 Too Many Requests: {\"jsonrpc\":\"2.0\",\"id\":16,\"error\":{\"code\":429,\"message\":\"Your app has exceeded its compute units per second capacity. If you have retries enabled, you can safely ignore this message. If not, check out https://docs.alchemy.com/reference/throughput\"}}"`

as for the `op-geth` it Is still starting to work and stopping the work, like so:

`INFO [04-19|18:25:15.343] Starting work on payload                 id=0x0249d3f9c66243e9
INFO [04-19|18:25:15.344] Updated payload                          id=0x0249d3f9c66243e9 number=54700 hash=fc59ae..ca4720 txs=1 withdrawals=0 gas=46889 fees=0 root=e409d6..9f44fb elapsed=1.034ms
INFO [04-19|18:25:15.345] Stopping work on payload                 id=0x0249d3f9c66243e9 reason=delivery elapsed=1
INFO [04-19|18:25:15.348] Imported new potential chain segment     number=54700 hash=fc59ae..ca4720 blocks=1 txs=1 mgas=0.047 elapsed=2.597ms     mgasps=18.049 age=1w5d12h snapdiffs=11.13KiB triedirty=0.00B
INFO [04-19|18:25:15.350] Chain head was updated                   number=54700 hash=fc59ae..ca4720 root=e409d6..9f44fb elapsed="794.878µs" age=1w5d12h
INFO [04-19|18:25:15.458] Starting work on payload                 id=0x029c1129e3bef19f
INFO [04-19|18:25:15.460] Updated payload                          id=0x029c1129e3bef19f number=54701 hash=69e721..3c2b18 txs=1 withdrawals=0 gas=46889 fees=0 root=ef4dd1..0cb2e6 elapsed="878.139
 s"
INFO [04-19|18:25:15.460] Stopping work on payload                 id=0x029c1129e3bef19f reason=delivery elapsed=1
INFO [04-19|18:25:15.463] Imported new potential chain segment     number=54701 hash=69e721..3c2b18 blocks=1 txs=1 mgas=0.047 elapsed=2.306ms     mgasps=20.326 age=1w5d12h snapdiffs=11.30KiB triedirty=0.00B
INFO [04-19|18:25:15.465] Chain head was updated                   number=54701 hash=69e721..3c2b18 root=ef4dd1..0cb2e6 elapsed="656.414µs" age=1w5d12h`

**Reply by yusufsuhair** [reply]
Hi, was the errors stated there sufficient for your debugging?

**Reply by opfocus** [reply]
op-node log  please   At present, it is just a normal prompt before synchronization.

**Comment by yusufsuhair** [comment]
`op-node` latest logs, its still keep moving, so these are what was able to fetch


WARN [04-21|11:22:32.708] tx in inbox with unauthorized submitter  addr=0x5C1736106082Aa6B3eC66FC04408aEEb4739B66c hash=1193ab..b214d6 err=nil
WARN [04-21|11:22:32.708] tx in inbox with unauthorized submitter  addr=0xfFfF1f69204C1c4c1d1cBc0bEd68778c903810e2 hash=91f00a..edf5c5 err=nil
WARN [04-21|11:22:32.709] tx in inbox with unauthorized submitter  addr=0x94133005fc9efEEB15C383244C58a989eFc71F04 hash=bbb8d2..fce1e5 err=nil
WARN [04-21|11:22:32.709] tx in inbox with unauthorized submitter  addr=0xf663792Be0EdD00AFFB8BBe4Ac6d8185efD5671d hash=f99391..241584 err=nil
WARN [04-21|11:22:32.709] tx in inbox with unauthorized submitter  addr=0xD837232D3cb8cE6a54e0dc10BFAe3D304004284e hash=6a3bc7..7b6ad8 err=nil
WARN [04-21|11:22:32.710] tx in inbox with unauthorized submitter  addr=0x0F82E82268FA5de5070A088e54eAbc2dec07D615 hash=8ae706..be8d3c err=nil
WARN [04-21|11:22:32.710] tx in inbox with unauthorized submitter  addr=0xA869aF2883DD6C881bC59594aDfB3e8b136321b4 hash=326ddc..09875d err=nil
WARN [04-21|11:22:32.710] tx in inbox with unauthorized submitter  addr=0x36c727A69F4c8b123E7A879E16b2eb79da357A3b hash=bb5a63..462ef3 err=nil
INFO [04-21|11:22:32.710] created new channel                      origin=75c771..dbff28:5745566 channel=f92d43..ea8eeb length=334   frame_number=0 is_last=true
INFO [04-21|11:22:32.710] Reading channel                          channel=f92d43..ea8eeb frames=1
INFO [04-21|11:22:32.711] Found next batch                         batch_type=SingularBatch batch_timestamp=1,713,698,452 parent_hash=524b7d..f1a702 batch_epoch=11267d..cecfd9:5745556 txs=0
INFO [04-21|11:22:32.711] generated attributes in payload queue    txs=1 timestamp=1,713,698,452
INFO [04-21|11:22:32.712] Sync progress                            reason="reconciled with L1"                   l2_finalized=a224e6..44d746:668707 l2_safe=db78ce..a7f87c:669146 l2_pending_safe=db78ce..a7f87c:669146 l2_unsafe=69e52b..1bdc76:669196 l2_time=1,713,698,552 l1_derived=75c771..dbff28:5745566
INFO [04-21|11:22:32.714] Found next batch                         batch_type=SingularBatch batch_timestamp=1,713,698,454 parent_hash=db78ce..a7f87c batch_epoch=11267d..cecfd9:5745556 txs=0
INFO [04-21|11:22:32.715] generated attributes in payload queue    txs=1 timestamp=1,713,698,454
INFO [04-21|11:22:32.716] Sync progress                            reason="reconciled with L1"                   l2_finalized=a224e6..44d746:668707 l2_safe=82df72..5247c7:669147 l2_pending_safe=82df72..5247c7:669147 l2_unsafe=69e52b..1bdc76:669196 l2_time=1,713,698,552 l1_derived=75c771..dbff28:5745566
INFO [04-21|11:22:32.718] Found next batch                         batch_type=SingularBatch batch_timestamp=1,713,698,456 parent_hash=82df72..5247c7 batch_epoch=11267d..cecfd9:5745556 txs=0
INFO [04-21|11:22:32.718] generated attributes in payload queue    txs=1 timestamp=1,713,698,456
INFO [04-21|11:22:32.720] Sync progress                            reason="reconciled with L1"                   l2_finalized=a224e6..44d746:668707 l2_safe=eee7d5..d08a03:669148 l2_pending_safe=eee7d5..d08a03:669148 l2_unsafe=69e52b..1bdc76:669196 l2_time=1,713,698,552 l1_derived=75c771..dbff28:5745566
INFO [04-21|11:22:32.721] Found next batch                         batch_type=SingularBatch batch_timestamp=1,713,698,458 parent_hash=eee7d5..d08a03 batch_epoch=598692..a24770:5745557 txs=0
INFO [04-21|11:22:32.722] generated attributes in payload queue    txs=1 timestamp=1,713,698,458
INFO [04-21|11:22:32.723] Sync progress                            reason="reconciled with L1"                   l2_finalized=a224e6..44d746:668707 l2_safe=255b82..f10d2e:669149 l2_pending_safe=255b82..f10d2e:669149 l2_unsafe=69e52b..1bdc76:669196 l2_time=1,713,698,552 l1_derived=75c771..dbff28:5745566
INFO [04-21|11:22:32.725] Found next batch                         batch_type=SingularBatch batch_timestamp=1,713,698,460 parent_hash=255b82..f10d2e batch_epoch=598692..a24770:5745557 txs=0
INFO [04-21|11:22:32.725] generated attributes in payload queue    txs=1 timestamp=1,713,698,460
INFO [04-21|11:22:32.726] Sync progress                            reason="reconciled with L1"                   l2_finalized=a224e6..44d746:668707 l2_safe=043889..7e7b13:669150 l2_pending_safe=043889..7e7b13:669150 l2_unsafe=69e52b..1bdc76:669196 l2_time=1,713,698,552 l1_derived=75c771..dbff28:5745566
INFO [04-21|11:22:32.728] Found next batch                         batch_type=SingularBatch batch_timestamp=1,713,698,462 parent_hash=043889..7e7b13 batch_epoch=598692..a24770:5745557 txs=0
INFO [04-21|11:22:32.728] generated attributes in payload queue    txs=1 timestamp=1,713,698,462
INFO [04-21|11:22:32.729] Sync progress                            reason="reconciled with L1"                   l2_finalized=a224e6..44d746:668707 l2_safe=631471..02c2c1:669151 l2_pending_safe=631471..02c2c1:669151 l2_unsafe=69e52b..1bdc76:669196 l2_time=1,713,698,552 l1_derived=75c771..dbff28:5745566
INFO [04-21|11:22:33.958] inserted block                           hash=f3fa16..41d027 number=669,197   state_root=d6a373..e9a921 timestamp=1,713,698,554 parent=69e52b..1bdc76        prev_randao=3181f2..fe9e87 fee_recipient=0x4200000000000000000000000000000000000011 txs=1 update_safe=false

**Reply by yusufsuhair** [reply]
okay i dont see any errors anymore

![image](https://github.com/ethereum-optimism/developers/assets/53525901/4f18e0ed-8b60-424a-a331-20de2cdd9882)

but i couldn't connect thru the metamask network. cannot add network

![image](https://github.com/ethereum-optimism/developers/assets/53525901/17ae6ee4-9e1d-4a5d-9acb-6604b1072b7f)

FYI, the chain id is from the `--networkid=42069` right?

./build/bin/geth \
  --datadir ./datadir \
  --http \
  --http.corsdomain="*" \
  --http.vhosts="*" \
  --http.addr=0.0.0.0 \
  --http.api=web3,debug,eth,txpool,net,engine \
  --ws \
  --ws.addr=0.0.0.0 \
  --ws.port=8546 \
  --ws.origins="*" \
  --ws.api=debug,eth,txpool,net,engine \
  --syncmode=full \
  --gcmode=archive \
  --nodiscover \
  --maxpeers=0 \
  --networkid=42069 \
  --authrpc.vhosts="*" \
  --authrpc.addr=0.0.0.0 \
  --authrpc.port=8551 \
  --authrpc.jwtsecret=./jwt.txt \
  --rollup.disabletxpoolgossip=true

**Reply by opfocus** [reply]
click peggle

**Reply by yusufsuhair** [reply]
nope, still not working

by being said there, its the chain ID that is not detected

![image](https://github.com/ethereum-optimism/developers/assets/53525901/56f68903-2e92-46f5-8d27-16ca260399d9)

**Reply by opfocus** [reply]
try http://your public ip:8545   or http://ejoe.xyz:8545

**Reply by yusufsuhair** [reply]
omg hey wow its now working!!

apaprently i cant do https here, that's weird,

so my next step, one day if I wanna do it in mainnet,

is there any proper documentation for that?

because once I got funded, I wanna go doing l2 rollup and do some bridging,

btw we can close this discussion already :))))))

**Reply by opfocus** [reply]
https://www.optimism.io/build    
Before you plan to deploy in a production environment, please contact the team for some advice.

**Reply by yusufsuhair** [reply]
got it

final inquiry

how can make it https abled?

bcz in mobile. i need it to be https

![image](https://github.com/ethereum-optimism/developers/assets/53525901/f058755f-3ac1-42b5-8776-98f93d59a0c0)

**Reply by opfocus** [reply]
I am not familiar with how to configure support for HTTPS access on the server (apparently requiring the installation of an SSL certificate), but I also believe that HTTPS access is essential.

**Reply by opfocus** [reply]
If you don't have any other questions, please mark this issue as answered. You can find me on Discord as simple8720. Wish you all the best

**Comment by yusufsuhair** [comment]
so basically i deployed there at https://ejoe.xyz

and this is the IP

![image](https://github.com/ethereum-optimism/developers/assets/53525901/c2ef4377-f458-43a0-95f6-ff79baafa65f)
